mti_nma/steps/nma/nma.py

```diff
@@ -38,42 +42,43 @@ class Nma(Step):
 
     def __init__(
         self,
-        direct_upstream_tasks=[Avgshape],
-        filepath_columns=["w_FilePath", "v_FilePath", "fig_FilePath"]
+        direct_upstream_tasks: Optional[List["Step"]] = [Avgshape],
+        filepath_columns=["w_FilePath", "v_FilePath", "vmag_FilePath", "fig_FilePath"]
     ):
         super().__init__(
             direct_upstream_tasks=direct_upstream_tasks,
             filepath_columns=filepath_columns
         )
 
     @log_run_params
-    def run(self, mode_list=list(range(6)), **kwargs):
+    def run(self, mode_list=list(range(6)), avg_df=None, path_blender=None, **kwargs):
+        """
+        This function will run normal mode analysis on an average mesh.
+        It will then create heatmaps of the average mesh for the desired set of modes,
+        where the coloring shows the relative amplitudes of vertex oscillation in 
+        the mode.
+
+        Parameters
+        ----------
+        mode_list: list
+            List of indices of modes to create heatmap files for
+
+        path_blender: str
+            Path to your local download of the Blender Application.
+            If on Mac, the default Blender Mac download location is used.
+        """
 
```

Comment:
Same here, recommend defaulting to upstream step CSV so that you can run in CLI.

Reply:
Awesome. Agreed. Will refer to the link you included in the other statement about this to implement for all steps (except the first)